pkg/quota_management/quota.go

```diff
@@ -0,0 +1,40 @@
+package quota_management
+
+import (
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/shared"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/shared/utils/arrays"
+)
+
+var defaultBillingModel = BillingModel{
+	ID:          "STANDARD",
```

Comment:
can we make the id lowercase to match what we have for other kinds of billing models?